_data/apps.yml

```diff
@@ -1331,3 +1331,17 @@
     - "reader"
     - "developer tool"
   icon: "jasper.png"
+-
+  name: "Vrap"
+  description: "360 media player"
+  website: "http://www.airpano.com/vrap.php?set_language=2"
+  repository: ""
+  keywords:
+    - "360 video"
+    - "360 player"
+    - "VR Player"
+    - "360 panorama player"
+    - "make 360 video"
+    - "krpano"
+    - "developer tool"
+  icon: "jasper.png"
```

Comment:
I uploaded icon png
not sure i did it correct

Thanks,
Andrey


On 22/06/16 18:00, Jessica Lord wrote:
>
> In _data/apps.yml 
> <https://github.com/electron/electron.atom.io/pull/354#discussion_r68081801>:
>
> > @@ -1331,3 +1331,17 @@
> >      - "reader"
> >      - "developer tool"
> >    icon: "jasper.png"
> > +-
> > +  name: "Vrap"
> > +  description: "360 media player"
> > +  website: "http://www.airpano.com/vrap.php?set_language=2"
> > +  repository: ""
> > +  keywords:
> > +    - "360 video"
> > +    - "360 player"
> > +    - "VR Player"
> > +    - "360 panorama player"
> > +    - "make 360 video"
> > +    - "krpano"
> > +    - "developer tool"
> > +  icon: "jasper.png"
>
> This is the icon for the previously listed project. You need to upload 
> and reference an icon for your project here. Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/electron/electron.atom.io/pull/354/files/07dec974d135a152ad3b6492a31539cce1f44ce4#r68081801>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AHxe4rT5AavlhYaDVKnxYqb2FnriFuc-ks5qOVwqgaJpZM4I7kA1>.
>